contracts/StakeHub.sol

```diff
@@ -133,6 +139,14 @@ contract StakeHub is SystemV2, Initializable, Protectable {
     // agent => validator operator address
     mapping(address => address) public agentToOperator;
 
+    // network related values //
+    // governance controlled maximum number of NodeIDs per validator (default is 5).
+    uint256 public maxNodeIDs = 5;
+
+    // mapping from a validator's address to an array of their registered NodeIDs,
+    // where each NodeID is stored as a fixed 32-byte value.
+    mapping(address => bytes32[]) private validatorNodeIDs;
```

Comment:
consensusToNodeIDs
follow the other names in this file, such as `consensusToOperator` or `consensusExpiration`